consensus/milestone_tracker/milestone_tracker.c

```diff
@@ -173,8 +173,9 @@ static void* milestone_validator(void* arg) {
       hash243_queue_pop(&mt->candidates);
       rw_lock_handle_unlock(&mt->candidates_lock);
       hash_pack_reset(&pack);
-      if (iota_tangle_transaction_load(mt->tangle, TRANSACTION_FIELD_HASH,
-                                       &hash, &pack) == RC_OK &&
+      if (iota_tangle_transaction_selective_load(
+              mt->tangle, TRANSACTION_FIELD_HASH, &hash, &pack,
+              MODEL_TRANSACTION_META_ALL) == RC_OK &&
```

Comment:
Note to myself: we actually only need the obsolete tag from this transaction so we might want to load only this field